src/cpu/core_dyn_x86/decoder.h

```diff
@@ -281,6 +282,16 @@ static void dyn_reduce_cycles(void) {
 	gen_dop_word_imm(DOP_SUB,true,DREG(CYCLES),decode.cycles);
 }
 
+static void dyn_save_vmware_relevant_regs(void) {
+	// VMware interface uses these registers for bidirectional
+	// communication with guest side tools, they have to be
+	// up to date when reading the magic IO port
+	gen_releasereg(DREG(EAX));
+	gen_releasereg(DREG(ECX));
+	gen_releasereg(DREG(EDX));
+	gen_releasereg(DREG(EBX));
+}
+
```

Comment:
(minor)
In new code and touched functions, we always replace `(void)` -> `()`.
(which also mirror how the function is called later on).

Reply:
Usually when doing only small changes I follow the conventions found in the file, especially if it is uniform. But OK, changed.